src/runtime/entries/netlify-edge.ts

```diff
@@ -20,7 +20,7 @@ export default async function (request: Request, _context) {
   })
 
   return new Response(r.body, {
-    headers: r.headers,
+    headers: r.headers as HeadersInit,
```

Comment:
I believe this was a mistake assuming the incoming request type of netlify edge was a standard Request.